app/renderer/js/utils/system-util.ts

```diff
@@ -32,14 +34,9 @@ export function getOS(): string {
 		return '';
 	}
 }
-
-export function setUserAgent(webViewUserAgent: string): void {
-	userAgent = `ZulipElectron/${app.getVersion()} ${webViewUserAgent}`;
-}
-
 export function getUserAgent(): string | null {
 	if (!userAgent) {
-		setUserAgent(ConfigUtil.getConfigItem('userAgent', null));
+		console.log('userAgent found null');
```

Comment:
Hmm, I don't think using a `console.log()` statement is a great idea - we're already logging the UA sent on the server side, right?
Maybe we can use a Sentry call here instead if we do client-side logs.

Reply:
@akashnimare what would you suggest ?